ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -1,18 +1,81 @@
+base_images:
+  base:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v22
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.9
 build_root:
-  project_image:
-    dockerfile_path: Dockerfile.ci
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v22
+images:
+- dockerfile_literal: FROM quay.io/ocs-dev/odf-operator:latest
+  to: odf-operator
+- dockerfile_path: Dockerfile.ci
+  from: base
+  to: odf-console
+- context_dir: openshift-ci/
+  dockerfile_path: index.Dockerfile
+  from: operator-index
+  to: noobaa-index
+operator:
+  bundles:
+  - as: fixed-dependencies
+    base_index: noobaa-index
+    context_dir: openshift-ci
+    dockerfile_path: bundle.Dockerfile
+  - as: odf-operator-master
+    base_index: ci-index-fixed-dependencies
+    context_dir: openshift-ci
+    dockerfile_path: ocsbundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/ocs-dev/odf-console:.*
+    with: odf-console
+  - pullspec: quay.io/ocs-dev/odf-operator:.*
+    with: odf-operator
 resources:
   '*':
     limits:
       memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
 tests:
 - as: lint
   commands: yarn install && yarn run lint
   container:
     from: src
+- as: odf-console-e2e-aws
+  optional: true
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-odf-operator-master
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-storage
+      OO_PACKAGE: odf-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: tests
+      commands: BRIDGE_KUBEADMIN_PASSWORD="$(cat /tmp/artifacts/installer/auth/kubeadmin-password)";
```

Comment:
We'll need to tweak the config here a bit (similar to https://github.com/openshift/release/blob/be73d9636886da0f5955a7712101d7ab8817fda3/ci-operator/config/openshift/console/openshift-console-master.yaml#L104) to allow the prow script injection for https://github.com/red-hat-storage/odf-console/pull/38 to work.